books/RayTracingTheNextWeek.html

```diff
@@ -2714,56 +2850,54 @@
 make an axis-aligned block primitive that holds 6 rectangles:
```

Comment:
The Cornell Box usually has two blocks in it. These are rotated relative to the walls. First, let’s create a function that returns a box, by creating a `hittable_list` that holds 6 rectangles:

Reply:
Reworded.